packages/trilinoscouplings/examples/scaling/example_Maxwell.cpp

```diff
@@ -149,7 +149,9 @@
 
 // MueLu
 #include <MueLu_RefMaxwell.hpp>
+#ifdef HAVE_MUELU_EPETRA
```

Comment:
@lucbv would it be more consistent to ifdef like below (line 229-ish) as 
`#if defined(HAVE_MUELU_EPETRA) and defined(HAVE_TRILINOSCOUPLINGS_MUELU`?
The second check may be redundant/unnecessary, but just thought I'd check, changes approved either way :)